src/transformers/models/mask2former/modeling_mask2former.py

```diff
@@ -2373,6 +2551,7 @@ def forward(
         mask_labels: Optional[List[Tensor]] = None,
         class_labels: Optional[List[Tensor]] = None,
         pixel_mask: Optional[Tensor] = None,
+        video_input: Optional[bool] = False,
```

Comment:
Please see my comment above, it's better to remove this to avoid backward compatibility issues
```suggestion
```